CPackConfig.txt

```diff
@@ -0,0 +1,79 @@
+set(CPACK_PACKAGE_NAME "capstone")
+set(CPACK_PACKAGE_VENDOR "Rot127")
+set(CPACK_PACKAGE_CONTACT "Rot127 <unisono@quyllur.org>")
+set(CPACK_PACKAGE_DESCRIPTION "Capstone is a lightweight multi-platform, multi-architecture disassembly framework. These are the development headers and libraries.\n Features:\n - Support hardware architectures: AArch64, ARM, Alpha, BPF, EVM, HPPA, LongArch, M680X, M68K, MOS65XX, Mips, PowerPC, RISCV, SH, Sparc, SystemZ, TMS320C64x, TriCore, WASM, x86, XCore, Xtensa.\n - Clean/simple/lightweight/intuitive architecture-neutral API.\n - Provide details on disassembled instructions (called \\\"decomposer\\\" by some others).\n - Provide some semantics of the disassembled instruction, such as list of implicit registers read & written.\n - Thread-safe by design.\n - Special support for embedding into firmware or OS kernel.\n - Distributed under the open source BSD license.")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Lightweight multi-architecture disassembly framework - devel files")
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://www.capstone-engine.org/")
+set(CPACK_STRIP_FILES false)
+
+# Set Debian-specific package variables
+set(CPACK_DEBIAN_PACKAGE_NAME "libcapstone-dev")
+set(CPACK_DEBIAN_PACKAGE_SOURCE "capstone")
+set(CPACK_DEBIAN_PACKAGE_VERSION "${PROJECT_VERSION}")
+set(CPACK_DEBIAN_PACKAGE_ORIGINAL_MAINTAINER "Debian Security Tools <team+pkg-security@tracker.debian.org>")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "libc6 (>= 2.2.5)")
+set(CPACK_DEBIAN_PACKAGE_SECTION "libdevel")
+set(CPACK_DEBIAN_PACKAGE_PRIORITY "optional")
+set(CPACK_DEBIAN_PACKAGE_MULTIARCH "same")
+
+# Determine architecture
+if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
+    set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "amd64")
+elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "i386" OR ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "i686")
+    set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "i386")
+elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm")
+    if(CMAKE_SIZE_OF_VOID_P EQUAL 4)
+        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "armhf")
+    else()
+        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "arm64")
+    endif()
+else()
+    set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE ${CMAKE_SYSTEM_PROCESSOR})
+endif()
+
+# Copy over Debian documentation
+install(FILES 
+    "${CMAKE_SOURCE_DIR}/ChangeLog" 
+    "${CMAKE_SOURCE_DIR}/CREDITS.TXT" 
+    "${CMAKE_SOURCE_DIR}/README.md" 
+    "${CMAKE_SOURCE_DIR}/SPONSORS.TXT"
+    DESTINATION "/usr/share/doc/libcapstone-dev"
```

Comment:
Looking at it a second time:
This is still a hard-coded path which might lead to problems on some systems.

I think you don't need to add the README, SPONSORS etc. to the package (or is there any requirement I am not aware?).
But apparently, you can add it using https://cmake.org/cmake/help/latest/module/CPack.html#variable:CPACK_RESOURCE_FILE_LICENSE.

The license is apparently added via [CPACK_RESOURCE_FILE_LICENSE](https://cmake.org/cmake/help/latest/module/CPack.html#variable:CPACK_RESOURCE_FILE_LICENSE)

Please check for the others.